servers/src/proto/journal/journal.proto

```diff
@@ -7,32 +7,31 @@ import "journal/lineage.proto";
 import "journal/raw_table.proto";
 
 // Wraps around all types of Tachyon journal messages.
-// next available id: 25
+// next available id: 24
 message JournalEntry {
   optional int64 sequence_number = 1;
   oneof entry {
     AddMountPointEntry add_mount_point = 2;
-    AsyncCompleteFileEntry async_complete_file = 3;
-    BlockContainerIdGeneratorEntry block_container_id_generator = 4;
-    BlockInfoEntry block_info = 5;
-    CompleteFileEntry complete_file = 6;
-    DeleteFileEntry delete_file = 7;
-    DeleteLineageEntry delete_lineage = 8;
-    DeleteMountPointEntry delete_mount_point = 9;
-    InodeDirectoryEntry inode_directory = 10;
-    InodeDirectoryIdGeneratorEntry inode_directory_id_generator = 11;
-    InodeFileEntry inode_file = 12;
-    InodeLastModificationTimeEntry inode_last_modification_time = 13;
-    LineageEntry lineage = 14;
-    LineageIdGeneratorEntry lineage_id_generator = 15;
-    PersistDirectoryEntry persist_directory = 16;
-    PersistFileEntry persist_file = 17;
-    PersistFilesEntry persist_files = 18;
-    PersistFilesRequestEntry persist_files_request = 19;
-    RawTableEntry raw_table = 20;
-    ReinitializeFileEntry reinitialize_file = 21;
-    RenameEntry rename = 22;
-    SetStateEntry set_state = 23;
-    UpdateMetadataEntry update_metadata = 24;
+    BlockContainerIdGeneratorEntry block_container_id_generator = 3;
```

Comment:
Reassigning the ids is fine before release because we don't mind breaking backcompat with unreleased versions. If we do alphabetical order, the ids would no longer be ordered. Ordering by ids doesn't get us much since humans will be interested in the name, not the id. Most of the proto API uses ids only internally.